test/test_io.py

```diff
@@ -35,6 +30,24 @@ def test_read_image_rs():
     assert cv_tensor.shape == [195, 258, 3]
 
     # convert to dlpack to import to torch
-    dlpack = K.cvtensor_to_dlpack(cv_tensor)
-    th_tensor = torch.utils.dlpack.from_dlpack(dlpack)
+    th_tensor = torch.utils.dlpack.from_dlpack(cv_tensor)
     assert th_tensor.shape == (195, 258, 3)
+
+
+def test_decompress():
+    # load an image with libjpeg-turbo
+    img_path: Path = DATA_DIR / "dog.jpeg"
+    tensor_rs = K.read_image_jpeg(str(img_path))
+    image = np.from_dlpack(tensor_rs)
```

Comment:
We can have an assert here? or maybe in another test with a small and simpler image (a black/white square)

Reply:
i will redo this tests possibly creating a synthetic image or something. It can be tricky thought becouse of the compression quality